ExpressAPI/db.js

```diff
@@ -0,0 +1,21 @@
+const mysql = require('mysql')
+const db = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: 'dorasdb'
+})
+
+db.connect(function(err) {
+    if (err) throw err
+    console.log('Connected to MySQL')
+})
+
+exports.getBand = function(callback) {
+    db.query(
+        "SELECT * FROM Band",
```

Comment:
remove `*`, explicitly define the column names! its bad practice

Reply:
@joshuahigginson